apl-updater/src/main/java/com/apollocurrency/aplwallet/apl/updater/UpdateTransactionVerifierImpl.java

```diff
@@ -18,8 +16,10 @@
 import com.apollocurrency.aplwallet.apl.util.Version;
 import org.slf4j.Logger;
 
-import java.util.regex.Pattern;
 import javax.inject.Inject;
+import java.util.regex.Pattern;
+
+import static org.slf4j.LoggerFactory.getLogger;
```

Comment:
I suppose it would be good to have a logging for ELSE cases, explaining WHY transaction is not accepted.
I mean ELSE LOsG for conditions like:
```
if (updateAttachment.getAppVersion().greaterThan.... ELSE  log.warn("Error reason...")
if (currentPlatform != null && currentPlatform.isAppropriate()... ELSE  log.warn("Error reason...")
```

Reply:
Transaction is accepted in any case, but for appropriate update transactions with critical level and matched platform spec we make additional actions such as calling supervisor (will be implemented in future)